resources/codegen_inputs/ift.rs

```diff
@@ -52,6 +65,12 @@ table PatchMapFormat1 {
 
   /// Patch format number for patches referenced by this mapping.
   patch_format: u8,
+
+  #[if_flag($field_flags, PatchMapFieldPresenceFlags::CFF_CHARSTRINGS_OFFSET)]
+  cff_charstrings_offset: u32,
```

Comment:
Yes, this is much nicer.

one final note is that it would be nice to document what this actually is? It's confusing to have a field named `_offset` that is not actually an offset to a subtable. Does this reference data here? Data in some other table?

(if it doesn't reference something that is an offset from this table I also wonder if maybe it wants to have a name that isn't `_offset`? it would be one of only (or very few) places in the spec where a field has a name that ends in 'offset` that is not an offset in the generally understood sense.)

Reply:
Added a comment, these are indeed offsets, but relative to the start of the CFF/CFF2 table.